release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1247,11 +1247,11 @@ All pod-to-pod network traffic between nodes on the OVN-Kubernetes cluster netwo
 
 For more information, see xref:../edge_computing/ztp-deploying-far-edge-sites.adoc#ztp-configuring-ipsec-using-ztp-and-siteconfig_ztp-deploying-far-edge-sites[Configuring IPsec encryption for {sno} clusters using {ztp} and SiteConfig resources].
 
-[id="ocp-4-16-hcp_{context}"]
-=== Hosted control planes
+//[id="ocp-4-16-hcp_{context}"]
+//=== Hosted control planes
 
-[id="ocp-4-16-insights-operator_{context}"]
-=== Insights Operator
+//[id="ocp-4-16-insights-operator_{context}"]
+//=== Insights Operator
```

Comment:
@cbippley Can you confirm that the following entry does not belong under this heading? 

The Insights Operator now collects instances outside of the openshift-monitoring of the following custom resources:

This is listed under Bug Fixes/Installer, but I'm not sure if it actually belongs there. It might actually belong under === Insights Operator

Reply:
Darragh is moving that one to Enhancements so he can un-comment out the heading if necessary. Good catch!